app/src/main/java/com/github/geohunt/app/ui/components/challengecreation/CreateChallengeViewModel.kt

```diff
@@ -28,106 +30,42 @@ class CreateChallengeViewModel(
     val displaySetting: Boolean,
 ) : ViewModel() {
 
-    enum class State {
-        AWAITING_LOCATION_PERMISSION,
-        AWAITING_CAMERA,
-        AWAITING_LOCATION,
-        READY_TO_CREATE,
-        CREATING
-    }
-
-    private val _location : MutableStateFlow<Location?> = MutableStateFlow(null)
-    val location : StateFlow<Location?> = _location
-
-    private val _expirationDate: MutableStateFlow<LocalDate?> = MutableStateFlow(null)
-    val expirationDate: StateFlow<LocalDate?> = _expirationDate
-
     private val _selectedDifficulty: MutableStateFlow<Challenge.Difficulty> =
-        MutableStateFlow(Challenge.Difficulty.MEDIUM)
+    MutableStateFlow(Challenge.Difficulty.MEDIUM)

```

Comment:
maybe put it in the same line or indent it?

Reply:
Oupsis